crates/bevy_input/src/input.rs

```diff
@@ -35,7 +35,7 @@ use bevy_ecs::schedule::State;
 /// * Call the [`Input::release`] method for each release event.
 /// * Call the [`Input::clear`] method at each frame start, before processing events.
 #[derive(Debug, Clone, Resource, Reflect)]
-#[reflect_value]
+#[reflect(Default)]
```

Comment:
Could you note this change inside a Migration Guide section in the PR description?